src/OneExplorer/OneExplorer.ts

```diff
@@ -111,6 +107,11 @@ class Node {
   get ext(): string {
     return path.extname(this.uri.fsPath);
   }
+
+  get typeAsString(): string {
```

Comment:
(7) Replace nodeTypeToStr with node.typeAsString getter

From [here](https://github.com/Samsung/ONE-vscode/pull/1020/files#r919809281), it is moved as a getter